src/components/Home/overviewData/LatestTreasuryOverview.tsx

```
@@ -143,10 +125,34 @@ const LatestTreasuryOverview = ({
 
+	useEffect(() => {
+		const fetchActiveBountyBalance = async () => {
+			if (!api || !apiReady) return;
+
+			try {
+				const bountyEntries = await api.query.bounties.bounties.entries();
+				let totalValue = 0;
+				bountyEntries.forEach(([_, bountyOption]) => {
+					if (bountyOption.isSome) {
+						const bountyData = bountyOption.unwrap().toJSON();
+						if (bountyData.value && !isNaN(Number(bountyData.value))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)